docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -33,6 +33,12 @@ $ sudo chmod +x /usr/local/bin/docker-machine-driver-kvm
 
 For more information, see the GitHub documentation of the link:https://github.com/dhiltgen/docker-machine-kvm#quick-start-instructions[docker machine KVM driver].
 
+KVM has a standing issue. You might face an error like
```

Comment:
Should be place at troubleshooting page? @LalatenduMohanty

Reply:
standing issue?